editor/plugins/org.fusesource.ide.wsdl2rest/.classpath

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8">
+		<attributes>
+			<attribute name="maven.pomderived" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry exported="true" kind="lib" path="lib/args4j-2.0.31.jar"/>
```

Comment:
this is still an impressive list of jars embedded in the plugin.
Do you have an idea of the size that it takes?
Do you have an idea how many of these dependencies are available on orbit or Jboss Tools Target Platform? (to evaluate the work to remove them)

Reply:
We are limited here by the wsdl2rest tool itself. I simply brought in the jars it needed to operate successfully. So no, I have not looked at the size. And no, I have no idea what is available in orbit or the target platform. I suspect the versions alone will cause us issues even if some of these jars are available in orbit. But we had similar issues when we wrapped the CXF code for SwitchYard several years ago and suspect that the same limitations apply.